src/conversion/model-to-view-converters.js

```diff
@@ -91,6 +90,34 @@ export function insertText() {
 	};
 }
 
+/**
+ * Function factory, creates a default model-to-view converter for node remove changes.
+ *
+ *		modelDispatcher.on( 'remove', remove() );
+ *
+ * @returns {Function} Remove event converter.
+ */
+export function remove() {
+	return ( evt, data, conversionApi ) => {
+		// Find view range start position by mapping model position at which the remove happened.
+		const viewStart = conversionApi.mapper.toViewPosition( data.position );
+
+		const modelEnd = data.position.getShiftedBy( data.length );
+		const viewEnd = conversionApi.mapper.toViewPosition( modelEnd, true );
```

Comment:
`const viewEnd = conversionApi.mapper.toViewPosition( modelEnd, { isPhantom: true } );` would be more readable.